indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -551,46 +548,123 @@ type("TimetableBlockWholeDayBase", ["TimetableBlockBase"],
      }
    );
 
+function drawBalloon (self, evt, editable) {
+    var timetableBlock = self.div.dom;
+    var entryId = self.eventData.scheduleEntryId ? self.eventData.scheduleEntryId : self.eventData.id.substring(1);
```

Comment:
out of curiosity, what's the first char of `self.eventData.id` you need to cut off?